### ISSUE TITLE
The WinForms memory viewer doesn't handle unaligned addresses

### ISSUE THREAD

**Issue by smx-smx** [uxmal/reko#516]
By entering a non-aligned (non multiple of 16) address in the address bar and hitting "Go",  the memory viewer will show "n" heading white spaces, where "n" is equal to addr % 16
![immagine](https://user-images.githubusercontent.com/1978844/34100658-7758dd1a-e3e3-11e7-9a0f-41a3477b16ee.png)

**Comment by chostelet** [comment]
Closed issue #577 is similar, probably related, but not exactly identical to this issue #516. As explained, and taking the example above, if one scrolls down in this view (E.g. using the mouse wheel) and then scrolls up back, the first line (with the spaces) won't be displayed again. Whatever "scroll up!" you attempt. You'll have to click on the Up-Arrow of the vertical scrolling bar to retrieve the first line. Curing issue #516 may correct issue #577, but who knows?

An other example:
![image](https://user-images.githubusercontent.com/7649238/36960683-5e4beffa-2048-11e8-807e-67cbcffa8773.png)

Addresses are correctly displayed (modulo 16) and synchronized (memory/disassembly).

Now let scroll-down with the mouse wheel (one step suffices), then up (as many steps as you wish). Result is:

![image](https://user-images.githubusercontent.com/7649238/36960797-e350bf1e-2048-11e8-9a4e-021a09832e2c.png)

To view again the first line, one has to click on the vertical scrolling bar.

**Comment by smx-smx** [comment]
Yes your issue is more related to the segment boundaries, but it's very likely related to the non 16 bytes aligned start of the segment.
The memory viewer apparently has a granularity of 16 bytes when scrolling so it will refuse to go up once it aligns to 16 bytes by scrolling up-down
